internal/venice-common/src/main/java/com/linkedin/davinci/schema/SchemaUtils.java

```diff
@@ -30,8 +30,14 @@ private SchemaUtils() {
    * @return Annotated value schema.
    */
   public static Schema annotateValueSchema(Schema schema) {
```

Comment:
Is changing just this method sufficient or should we introduce a wrapper method to be used for all the read-only parsing actions? IIRC I see a lot of external components call `AvroSchemaParseUtils.parseSchemaFromJSONStrictValidation` directly too or do you plan to change them individually? Personally I prefer to have a centralized method where we can control the parsing behavior. e.g All the read-only use cases will use `parseSchemaFromJSONLooseValidation` and all the use cases that can potentially add new schema (write) will continue to use `parseSchemaFromJSONStrictValidation`